workshops/personal_website/README.md

```diff
@@ -164,330 +130,221 @@ Every HTML file follows this template:
 </html>
 ```
 
-Let's put this template inside of our HTML file:
+This structure is common to all HTML pages. In fact, you can take a look for
+yourself! Just right click on any web page, including this one, and click "View
+source" to see what's going on behind the scenes. You'll find each of these
+elements on every page-- the doctype, and an HTML tag wrapped around a head and
+body.
+
+Before proceeding, we'll briefly go over what this template means. HTML works
+by storing information inside tags. `<html></html>` is an example of one such
+tag. Inside `<html></html>`, we've placed two other sets of tags:
+`<head></head>` (which wraps around the "head") and `<body></body>` (which
+wraps around the "body"). The body holds everything you would see in the actual
+tab/window when you open the page, while the head conveys information about the
+page to the browser.
+`<!DOCTYPE html>` tells the browser what version of HTML to expect. Since it is
+a language, HTML is constantly growing and updating, so there are multiple
+versions. In our case, we are going to use HTML5, the latest version.
```

Comment:
Can probably be simplified to say "`<!DOCTYPE html>` tells the browser to use the latest version of HTML. We have to include this because some browsers (like Internet Explorer) don't support all of the latest features in HTML."